index.html

```diff
@@ -9,40 +9,71 @@
 
     <body>
 
-        <maintext class="box" action="index.html" method="post">
-        <h1>Anti-Xeno Synthisis Calculator</h1> 
-
-        <input type="text" id="heatsink" placeholder="0">
-        <label>Heatsinks</label> <br>
-        <input type="text" id="flak" placeholder="0">
-        <label>Remote Release Flak</label> <br>
-        <input type="text" id="gauss" placeholder="0">
-        <label>Gauss Cannon</label> <br>
-        <input type="text" id="gaussPrem" placeholder="0">
-        <label>Gauss Cannon Prem Ammo</label> <br>
-        <input type="text" id="shard" placeholder="0">
-        <label>Shard Cannon</label> <br>
-        <input type="text" id="shardPrem" placeholder="0">
-        <label>Shard Cannon Prem Ammo</label> <br>
-        <input type="text" id="limpets" placeholder="0">
-        <label>Limpets</label> <br>
-        <input type="text" id="afmu" placeholder="0">
-        <label>AFMU Refills</label> <br>
-        <input type="text" id="lifeSupport" placeholder="0">
-        <label>Life Support</label> <br>
-
-        <div class="button-wrapper">
-            <button class="button" onclick="copyMatsList()">Copy Mats List</button>
-        </div>
-
-        </maintext>
-
-        <matsDisplay class="box-2">
-            <div class="text-wrap">
-            <larger class="bigger">Material List: <br> </larger>
-                <div id="textbox" class="textbox"></div>
-            </div>
-        </matsDisplay>
+        <grid id="grid-container">

```

Comment:
![image](https://user-images.githubusercontent.com/35440982/120051538-06f8f500-bfef-11eb-8df3-995355c7f5cd.png)

The wrapping and loss of text are annoying. You have plenty of screen space, use it

Reply:
This also applies to the material list

Reply:
Won't fix text wrapping. Will fix cutoff

Reply:
Cutoff fixed.